### PR TITLE
Integration of telegram bot

### PR DESCRIPTION
**Pull request by vladimirdonev**
Creted Integration with telegram

### CODE REVIEW

**Comment by VeselinMetodiev** [comment]
Ok.

File: ProjectB/ProjectB/Properties/launchSettings.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "anonymousAuthentication": true,
     "iisExpress": {
       "applicationUrl": "http://localhost:43954",
-      "sslPort": 44365
+      "sslPort": 0
```
Comment (it-kyrylo) [inline ProjectB/ProjectB/Properties/launchSettings.json:8]: you do need this changes

File: ProjectB/ProjectB/ProjectB.csproj [hunk 2]
```diff
@@ -6,10 +6,14 @@
 
   <ItemGroup>
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="8.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="5.0.9" />
     <PackageReference Include="Refit" Version="6.0.94" />
     <PackageReference Include="Refit.HttpClientFactory" Version="6.0.94" />
     <PackageReference Include="Refit.Newtonsoft.Json" Version="6.0.94" />
+    <PackageReference Include="SimpleInjector.Integration.AspNetCore" Version="5.3.0" />
```
Comment (it-kyrylo) [inline ProjectB/ProjectB/ProjectB.csproj]: Remove SimpleInjector

**Review by it-kyrylo — APPROVED** [review]

**Comment by it-kyrylo** [comment]
Please fix my comments before merging them to master

**Review by it-kyrylo — APPROVED** [review]

**Pull request merged**